### PR TITLE
fix: change fluentD configs for tail logs

### PR DESCRIPTION
**Pull request by ohnickmoy**
Configuration Pull Request
---

Change fluentD configurations to tail a rotating file handler

Mounts the rotating file handler on sandbox onto the container 

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -802,12 +802,10 @@ function provision_fluentd() {
     echo "fluentd_config=/var/tmp/fluentd.conf"
     echo "cat << 'EOF' > \$fluentd_config
     <source>
-        @type syslog
-        port 5140
-        bind 0.0.0.0
+        @type tail
+        path /var/tmp/tracking_logs.log
+        pos_file /var/log/tracking_logs.pos
```
Comment (alangsto) [inline util/jenkins/ansible-provision.sh:807]: Is any additional configuration needed for the file rotation? Specifically thinking about https://docs.fluentd.org/input/tail#rotate_wait

Comment (ohnickmoy) [reply util/jenkins/ansible-provision.sh:807]: It depends on how much we're comfortable with the delay and i suppose how much these tracking logs get appended to. default is 5, but would say...idk 20 seconds be a big enough window for rotate_wait?

The other settings that come to mind is probably [encoding](https://docs.fluentd.org/input/tail#encoding-from_encoding)  (by default its ascii-8bit)

Comment (ohnickmoy) [reply util/jenkins/ansible-provision.sh:807]: @alangsto, spoke with @NIXKnight in sync meeting and he thinks 10 second window might be good. Too small of a window?

**Review by adzuci — APPROVED** [review]
I think I understand this and it looks good to merge.

**Pull request merged**